flowetl/etl/tests/test_dummy_callables.py

```diff
@@ -3,52 +3,57 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 # -*- coding: utf-8 -*-
+"""
+Tests that the testing dummy calables behave as expected
+"""
+from unittest.mock import patch
```

Comment:
Why `patch` and not pytest's `monkeypatch`, out of interest?

Reply:
No reason - I know how to use that one

Reply:
@greenape Out of interest, do you know what the difference is between the two? When is `monkeypatch` preferable, or are they mostly equivalent?

Reply:
I think they're pretty interchangeable to be honest - with monkeypatch, you don't need the context handler because it'll undo everything once the test finishes, and there's a specific `monkeypatch.setenv` method. I was just curious.